### ISSUE TITLE
Zotino & Fastino FP connector in DIOT

### ISSUE THREAD

**Issue by marmeladapk** [sinara-hw/meta#89]
We're thinking about changing front panel connector on Fastino and Zotino in DIOT versions to mini SAS (the same ones that are in Shuttler, maybe we could even use Shuttler frontend). They have much lower crosstalk that has been problematic in case of Fastino and even long cables aren't expensive. That way we could avoid adapters hanging on the FP. This would mean that accompanying adapters (HD68 and possibly SMA/BNC) would also be changed for DIOT.

**Comment by kaolpr** [comment]
How do you want to place adapters / mezzanines (e.g. HVAMP32)? External non-DIOT crate?

**Comment by gkasprow** [comment]
HVAMP32 fits nicely within 6HP DIOT Panel
![image](https://github.com/sinara-hw/meta/assets/4325054/e29ab3b2-16d9-444f-bd87-0a29f11d6192)

**Comment by gkasprow** [comment]
In the case of mini SAS plugs I designed an adapter that outputs 8 u.fl to SMA pigtails
![image](https://github.com/sinara-hw/meta/assets/4325054/d0c12a2f-71e2-4221-8725-1bcd474a0594)

**Comment by gkasprow** [comment]
They nearly fit. One would have to shift the HVAMP connectors 1mm to the back of the board
![image](https://github.com/sinara-hw/meta/assets/4325054/145f83da-8167-4e6f-83c1-fb140e5343a7)

**Comment by kaolpr** [comment]
Perfect! I don't think there are many HVAMPs out there right now and anyway they'll go combined.

However, I think that some more user-resistant adapter (e.g. a box connected with cable, like Shuttler AFE but with SMA) would be nice. It could be used separately from the pig-tail adapter which could be nice for some quick testing.

**Comment by gkasprow** [comment]
Yep. We can make the same kind of adapter that was made for HD68. I mean one with BNCs and MCXs.

**Comment by gkasprow** [comment]
When we replace 3M four wall headers with standard gold-pins, all will fit. 
One can fit existing and new HVAMP32  and also use ribbon cables in EEM mode.
![image](https://github.com/sinara-hw/meta/assets/4325054/f17113b2-8f81-4578-b3f8-466c477b17ba)

**Comment by kaolpr** [comment]
It looks like this solution will somehow compromise ribbon usage as with goldpins it will be possible to connect it the wrong way. Maybe it's better to evaluate how many HVAMPs are there and see if we really need to care?

**Comment by gkasprow** [comment]
We make new Fastino release to fit new, longer HVAMP :)

**Comment by gkasprow** [comment]
We can also shift the connectors and loose compatibility with the first batch of HVAMPs

**Comment by kaolpr** [comment]
I believe this is not a big problem.

**Comment by gkasprow** [comment]
pls make sure we have same pinout of SPI interface in Shuttler
